### DISCUSSION TITLE
Add extrude lenght to measurement value.

### DISCUSSION THREAD

**Discussion by Hipshot-zz** [Garux/netradiant-custom#177] [Ideas]
Maybe the extra or added size of the brush can be added to the value so you know how much you expand with? Maybe like something in the image, where 32 was the original size and 88 is how much bigger it became?

Like in the blue value here blow:

![image 1](https://github.com/Garux/netradiant-custom/assets/178820/9e254a39-a26e-4dd5-8c68-27aab83b4622)

**Comment by Garux** [comment]
This could be useful to some degree. For now i'd prefer to keep simple display of selection size.
